hack/generate-manifest.sh

```diff
@@ -314,6 +320,12 @@ if $FLEXIBLE_IPAM; then
     sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*noSNAT[[:space:]]*:[[:space:]]*[a-z]+[[:space:]]*$/noSNAT: true/" antrea-agent.conf
 fi
 
+if $MULTICAST; then
+    sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*trafficEncapMode[[:space:]]*:[[:space:]]*[a-z]+[[:space:]]*$/trafficEncapMode: noEncap/" antrea-agent.conf
+    sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*Multicast[[:space:]]*:[[:space:]]*[a-z]+[[:space:]]*$/  Multicast: true/" antrea-agent.conf
+    sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*externalMulticastInterfaces[[:space:]]*:[[:space:]]*[a-z]+[[:space:]]*$/externalMulticastInterfaces: [ens192,ens224]/" antrea-agent.conf
```

Comment:
Please update `externalMulticastInterfaces` to `multicastInterfaces` in the antrea-agent(verify the sed command before commit, simply change `externalMulticastInterfaces` to `multicastInterfaces`) won't work.

Reply:
In antrea conf https://github.com/antrea-io/antrea/blob/23de6933f67ad309d11ff43768c611180f78e6da/build/yamls/antrea.yml
#multicastInterfaces: []

Reply:
reminder

Reply:
Please update the sed script for changing `multicastInterfaces`. Your code does not work because it needs to match `[interface,  interface]`. Please test this one and update your commit.  @XinShuYang 
```
    sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*multicastInterfaces[[:space:]]*:[[:space:]]*\[([a-zA-Z0-9]*,[[:space:]]*)*[a-zA-Z0-9]*\][[:space:]]*$/multicastInterfaces: [ens224]/" antrea-agent.conf

```

Reply:
Updated

Reply:
Updated, thanks!